packages/yarnpkg-core/sources/formatUtils.ts

```diff
@@ -301,6 +302,25 @@ const transforms = {
       return text;
     },
   }),
+
+  [Type.MARKDOWN_INLINE]: validateTransform({
+    pretty: (configuration: Configuration, text: string) => {
```

Comment:
Note that this implementation is different from `formatMarkdownish`, which would keep the `**` surrounding embolddened text, and add a trailing newline.